Mono.Debugging.Soft/SoftDebuggerAdaptor.cs

```diff
@@ -982,9 +996,28 @@ static T FindByName<T> (IEnumerable<T> items, Func<T,string> getName, string nam
 
 			return best;
 		}
-		
+
 		protected override IEnumerable<ValueReference> GetMembers (EvaluationContext ctx, object t, object co, BindingFlags bindingFlags)
 		{
+			return GetMembers (ctx, null, t, co, bindingFlags);
+		}
+
+		string [] GetTupleElementNames (IObjectSource source, EvaluationContext ctx)
+		{
+			switch (source) {
+			case FieldValueReference field:
+				return field.GetTupleElementNames ();
+			case PropertyValueReference prop:
+				return prop.GetTupleElementNames ();
```

Comment:
If I'm following the code correctly, this will be called for every GetMember call on a property, and this ends calling TupleElementNamesPrivate in PortablePdbData, which opens and reads the pdb to get the tuple information. This seems to be a slow operation to do on every call.